devlab/generate_load.py

```diff
@@ -750,6 +750,19 @@ def create_invalid_cinder_objects(self):
                 display_name=invalid_volume_tmlt % st)
             self.cinderclient.volumes.reset_state(vol, state=st)
 
+    def create_swirt_container_and_objects(self):
+        for cont in self.config.swift_containers:
+            self.put_swift_container(cont['name'])
+            for obj in cont['objects']:
+                size = obj['size_in_MB']
+                self.put_swift_object(cont['name'],
+                                      obj['name'],
+                                      os.urandom(1024*1024*size))
```

Comment:
Tests may actually need to verify contents of a swift object, so I'd suggest that you put a value which can be then checked (it can be both random or not, depends on how you implement it).